src/Application/Games/Commands/UpdateGameUsersCommand.cs

```diff
@@ -90,6 +110,17 @@ await _db.EquippedItems
             return charactersById;
         }
 
+        private async Task<IdempotencyKey?> LoadIdempotencyKey(Guid key, CancellationToken cancellationToken)
+        {
+            var idempotencyKey = await _db.IdempotencyKeys
+                .Where(ik => ik.Key == key.ToString())
+                .FirstOrDefaultAsync(cancellationToken);
+
+            return idempotencyKey;
+        }
+

```

Comment:
Please move this to the main function. There is no reuse of this method. It just makes it harder to spot issues in the main method.